ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -246,6 +246,78 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-disruptive,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws-fips
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-master-e2e-aws-fips
+    optional: true
+    rerun_command: /test e2e-aws-fips
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --branch=master
+        - --give-pr-author-access-to-namespace=true
+        - --org=openshift
+        - --repo=installer
+        - --resolver-address=http://ci-operator-configresolver-ci.svc.ci.openshift.org
+        - --secret-dir=/usr/local/e2e-aws-fips-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws-fips
+        - --template=/usr/local/e2e-aws-fips
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-installer-master.yaml
+              name: ci-operator-master-configs
+        - name: ENABLE_FIPS
+          value: "true"
```

Comment:
manually edited this job after generation to add the ENABLE_FIPS - there doesn't seem to be a way to pass this in
ci-op/config, I will check though.

Reply:
> I will check though.
@stevekuznetsov I can't pass this value in the config file rather than a manual edit to the job? : )

Reply:
when I run `make jobs` to fix `/test generated-config` this ENABLE_FIPS disappears, so not sure how to set this and satisfy that check.

Reply:
removed the entry from ci-op/config/openshift/origin|installer after generating the  job